### ISSUE TITLE
[FSR Wizard] Formative study planning

### ISSUE THREAD

**Issue by charlottecesana** [department-of-veterans-affairs/va.gov-team#71452]
## Issue Description
_What details are necessary for understanding the specific work or request tracked by this issue?_

The UX team is planning to run a lightweight formative Wizard study with Veterans. 
We will be using a mid-fidelity prototype. We are seeking to interview 5-6 Veterans in 30 - 50 minute sessions. 

---
## Tasks
_What work is necessary for this story to be completed?_

[STEP 1 of 3]
- [x]  UX team writes research plan, conversation guide, and recruiting criteria
- [x]  UX team posts research plan, conversation guide, and recruiting criteria on Github

[STEP 2 of 3]
- [x] UX Research obtains platform collaboration research cycle approval (2-3 days)
- [ ] Note taking spreadsheet and Mural created

[STEP 3 of 3]
- [ ] Host a pilot session
- [ ] Coordinate with Perigean (1 week)

## Acceptance Criteria
 _What will be created or happen as a result of this story?_
- [ ] Research plan
- [ ] Conversation guide
- [ ] Note taking spreadsheet and Mural created
- [ ] Pilot session has taken place
- [ ] Research participants confirmed
---

**Comment by HeatherRienks** [comment]
This ticket blocked by CAIA feedback.

**Comment by mgleo** [comment]
We will be meeting with CAIA tomorrow 01/17

**Comment by HeatherRienks** [comment]
Per Sprint Planning,will break this ticket up

**Comment by HeatherRienks** [comment]
As of March 1, 2024, Lily Strelich from CAIA responded in Slack that:
_"we’ve been looking at where to place content from the wizard. Let me check in on our end and then I can message Tom and Heather with our availability. Thanks!"_

**Comment by HeatherRienks** [comment]
The CAIA team is working on a draft Static page in place of the Wizard. The CAIA ticket is: #73909.

@Tom-Davis -do we want to close this ticket?

**Comment by HeatherRienks** [comment]
Per Design sync, we are closing this ticket and will create a new ticket outlining the revised scope of work.
